features/steps/command/run.feature

```diff
@@ -20,3 +20,31 @@ Feature: Run commands
     When I run `cucumber`
     Then the features should all pass
 
+  Scenario: Activate desired announcers when running command fails
+    Given an executable named "bin/cli" with:
+    """
+    #!/bin/bash
+    echo "Hello, I'm STDOUT"
+    exit 1
+    """
+    And a file named "features/run.feature" with:
+    """
+    Feature: Run it
+      Scenario: Run command
+        When I successfully run `cli`
+    """
+    And a file named "features/step_definitions.rb" with:
+    """
+    Aruba.configure do |config|
```

Comment:
Mmmh... I would use a `Before`-hook here. The `configure`-syntax should go to `features/support/aruba.rb`. But be careful. This file already exists in the default-app.